api/client/client_test.go

```diff
@@ -69,6 +70,16 @@ func TestAuthSign(t *testing.T) {
 	}
 }
 
+func TestBundleAuthSign(t *testing.T) {
```

Comment:
I realize no test in this file has any comment describing the test case, but maybe we should add one :)

Reply:
The tests are not ideal indeed. I'll keep it at that minimum for now and hopefully find the time to improve them all